analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -445,7 +388,7 @@ def get_query(source: ObjectStream) -> ObjectStream:
 filemeta = run.preprocess(fileset, treename=treename)  # pre-processing
 
 t0 = time.monotonic()
-all_histograms, metrics = run(fileset, treename, processor_instance=TtbarAnalysis(DISABLE_PROCESSING, IO_FILE_PERCENT))  # processing
+all_histograms, metrics = run(fileset, treename, processor_instance=TtbarAnalysis(config["benchmarking"]["DISABLE_PROCESSING"], config["benchmarking"]["IO_BRANCHES"][config["benchmarking"]["IO_FILE_PERCENT"]]))  # processing
```

Comment:
I'm guessing this will be too long for a cell, let's try to break these super long lines up to make them easier to read (also affects the notebook, scrolling is not super convenient). Perhaps break at the last comma. We're not enforcing strict formatting currently (should eventually probably just adopt `black`) but I've been trying to at least make lines not too super long.